### PR TITLE
chore: update `wallet.rs`

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

Need some fixes included in this version.

## Changelog

```
- Increment the RC version to 20
- Added async/await to profileManager.destroy()
- Remove redundant typecast
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

Tested general wallet functionality, including Governance actions.

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/lib/core/ledger/constants/use-ledger-simulator.constant.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const USE_LEDGER_SIMULATOR = false
+export const USE_LEDGER_SIMULATOR = true
```
Comment (MarkNerdi) [inline packages/shared/lib/core/ledger/constants/use-ledger-simulator.constant.ts:1]: i think we want this disabled, right?

**Review by MarkNerdi — COMMENTED** [review]
1 small comment.
And because of the awaiting of the profile manager, it takes a long time to log out. Maybe we should at least add some loading spinner on the logout button

File: packages/shared/lib/core/ledger/constants/use-ledger-simulator.constant.ts (hunk 1 above)
Comment (maxwellmattryan) [reply packages/shared/lib/core/ledger/constants/use-ledger-simulator.constant.ts:1]: Yes, did you disable for testing? @Tuditi

**Comment by Tuditi** [comment]
> 1 small comment. And because of the awaiting of the profile manager, it takes a long time to log out. Maybe we should at least add some loading spinner on the logout button

Yeah, that's a bit annoying. Based on our discussions we will have to set a `databaseUnlocked` property on the profile store and check that before logging in. This way we don't have to add a loading screen on logout. This will come in a separate PR.

**Review by MarkNerdi — APPROVED** [review]
LGTM then :fire:

**Pull request merged**